### PR TITLE
fixing the editing dropdowns

### PR DESCRIPTION
**Pull request by neroliang**
# Acceptance Criteria
Link check rule edit pages load and update correctly

# Background

http://harvester.dnz0a.digitalnz.org/production/link_check_rules/5244bed0147d7b8283000098/edit
- notice the top dropdown does not load correctly
- unticking the active box is not being saved

*Rule edit page:*
http://harvester.dnz0a.digitalnz.org/production/link_check_rules

 Updated gems for ruby 2.3.0. Made changes for new version of Devise.
# Checklist
- [ ] Code is understandable without Dev
- [ ] Acceptance criteria, what was changed, and why it was changed (If applicable) on Pull / Merge Request
- [ ] All Tests are Passing
- [ ] Code Coverage goes up
- [ ] All new methods have a relevant spec
- [ ] Methods have a single responsibility (Where applicable)
- [ ] Linters Pass
- [ ] ‘Yard’ style comments on methods and classes (Where applicable)

---
This change is [Reviewable](https://reviewable.io/reviews/digitalnz/supplejack_manager/13)

### CODE REVIEW

**Comment by EdwinRozario** [comment]
👍

File: app/controllers/link_check_rules_controller.rb [hunk 1]
```diff
@@ -27,7 +27,11 @@ def new
   end
 
   def edit
-    @partners = Partner.where(:id.in => current_user.manage_partners).asc(:name)
+    if current_user.admin?
```
Comment (neroliang) [inline app/controllers/link_check_rules_controller.rb]: Good points.

File: app/controllers/link_check_rules_controller.rb [hunk 2]
```diff
@@ -27,7 +27,11 @@ def new
   end
 
   def edit
-    @partners = Partner.where(:id.in => current_user.manage_partners).asc(:name)
+    @partners = if current_user.admin?
```
Comment (DanHenton) [inline app/controllers/link_check_rules_controller.rb]: ternary statement or a helper method might be nice for this one. But all 3 options are okay i just think they are abit cleaner

**Comment by DanHenton** [comment]
👍

**Pull request merged**